[2025-02-24T04:47:16.964+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-24T04:47:17.007+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.great_func scheduled__2025-02-05T00:00:00+00:00 [queued]>
[2025-02-24T04:47:17.026+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.great_func scheduled__2025-02-05T00:00:00+00:00 [queued]>
[2025-02-24T04:47:17.027+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-02-24T04:47:17.054+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): great_func> on 2025-02-05 00:00:00+00:00
[2025-02-24T04:47:17.066+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1016) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-24T04:47:17.068+0000] {standard_task_runner.py:72} INFO - Started process 1033 to run task
[2025-02-24T04:47:17.072+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'great_func', 'scheduled__2025-02-05T00:00:00+00:00', '--job-id', '66', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dags.py', '--cfg-path', '/tmp/tmpbse_xx9h']
[2025-02-24T04:47:17.075+0000] {standard_task_runner.py:105} INFO - Job 66: Subtask great_func
[2025-02-24T04:47:17.161+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.great_func scheduled__2025-02-05T00:00:00+00:00 [running]> on host 79e0cb3ed452
[2025-02-24T04:47:17.274+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='gzang' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='great_func' AIRFLOW_CTX_EXECUTION_DATE='2025-02-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-05T00:00:00+00:00'
[2025-02-24T04:47:17.277+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-24T04:47:17.309+0000] {logging_mixin.py:190} INFO - *** chào Thùy, i am 1 days old!!!
[2025-02-24T04:47:17.311+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-24T04:47:17.323+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-24T04:47:17.324+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=great_func, run_id=scheduled__2025-02-05T00:00:00+00:00, execution_date=20250205T000000, start_date=20250224T044717, end_date=20250224T044717
[2025-02-24T04:47:17.531+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-24T04:47:17.553+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-24T04:47:17.555+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
